sdk/python/feast/feature_store.py

```diff
@@ -1297,8 +1299,8 @@ def get_online_features(
                 if isinstance(features, FeatureService)
                 else None,
                 entities=columnar,
+                request_data={},  # ToDo: add request data parameter to public API
```

Comment:
nit: for consistency can you do `TODO`

ideally also add your username if you intend on doing this in a followup